frontend/components/surplus/CollectSurplusAuctionPanel.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>
+            {{ currentStateAndTitle.title }}
+            <time-till v-if="auction.state === 'have-bids'" :date="auction.earliestEndDate" />
+        </template>
+        <TextBlock v-if="auction.state === 'ready-for-collection'">
```

Comment:
I would show this text for the collected auction state as well instead of the other text

Reply:
I am not sure what you mean with this. Which text should I display/change? 😞